Indicators/PivotPointsHighLow.cs

```diff
@@ -53,8 +54,9 @@ public class PivotPointsHighLow : IndicatorBase<IBaseDataBar>, IIndicatorWarmUpP
         /// </summary>
         /// <param name="surroundingBarsCount">The length parameter here defines the number of surrounding bars that we compare against the current bar high and lows for the max/min </param>
         /// <param name="lastStoredValues">The number of last stored indicator values</param>
-        public PivotPointsHighLow(int surroundingBarsCount, int lastStoredValues = 100)
-            : this($"PivotPointsHighLow({surroundingBarsCount})", surroundingBarsCount, surroundingBarsCount, lastStoredValues)
+        /// <param name="strictChecking">If true, uses strict inequality for pivot detection. If false, allows ties (central value can equal neighbors)</param>
+        public PivotPointsHighLow(int surroundingBarsCount, int lastStoredValues = 100, bool strictChecking = true)
```

Comment:
These new constructor changes are not compiling, it causes ambiguous errors, suggest just keeping this new argument in the bigger constructors, given it's more a use custom/specific use case